src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -811,6 +811,11 @@ internal static void ValidateFunction(FunctionDescriptor function, Dictionary<st
 
                 httpFunctions.Add(function.Name, httpTrigger);
             }
+            if (environment != null && environment.IsFlexConsumptionSku()
```

Comment:
Please remove this `environment != null` check so we're enforcing that callers pass in a valid environment. Going forward for this and potentially other features in the future, we're doing environmental/sku based checks, so we always need an environment.

Reply:
Fixed